src/main/java/com/google/devtools/build/lib/skyframe/ConfiguredTargetFunction.java

```diff
@@ -485,11 +485,11 @@ public SkyValue compute(SkyKey key, Environment env) throws ConfiguredTargetFunc
             configuration.getFragmentsMap().keySet(),
             BuildOptions.diffForReconstruction(defaultBuildOptions, toolchainOptions));
 
-    Map<String, UnloadedToolchainContextKey> unloadedToolchainContextKeys = new HashMap<>();
+    Map<String, ToolchainContextKey> unloadedToolchainContextKeys = new HashMap<>();
```

Comment:
or maybe this PR is the right time to change these var names? (:

Reply:
Fixed in #11336.